luatest/comparator.lua

```diff
@@ -70,6 +70,31 @@ function comparator.is_subset(actual, expected)
     return #expected_array - found_count
 end
 
+-- Checks that any element from the actual is an element from the expected.
+function comparator.any(actual, expected)
+    if (type(actual) ~= 'table') or (type(expected) ~= 'table') then
+        return false
+    end
+
+    local expected_array = {}
+    local expected_casted = {}
+    for _, v in pairs(expected) do
+        table.insert(expected_array, v)
+    end
```

Comment:
Why do you convert `expected` to an array? In `is_subset` this is used to return the number of elements that are present in `expected` but not in `actual` but here don't account it.

`comparator.cast` used below doesn't seem to be necessary, either.

Reply:
Thanks, fixed.